### PR TITLE
Update to 1.1.3

### PR DESCRIPTION
**Pull request by kpiwko**
./gradlew --stacktrace :tools:test-extension:client:assemble generates upte in build/distribution subfolder

./gradlew --stacktrace :tools:test-extension:server:war generated server war to be deployed next to ag-push.war

Note this changes are against EAP 6.4/JBossAS7 stream related testing. Also dropped OpenShift v2 and quickstart testing as they are no longer related.

### CODE REVIEW

**Comment by kpiwko** [comment]
@matzew fyi

File: tools/test-extension/client/src/main/java/org/jboss/aerogear/test/cli/AppCreateCommand.java [hunk 1]
```diff
@@ -1,128 +0,0 @@
-package org.jboss.aerogear.test.cli;
-
-import com.jayway.restassured.RestAssured;
-import io.airlift.airline.Command;
-import io.airlift.airline.Option;
-import org.arquillian.spacelift.Spacelift;
-import org.arquillian.spacelift.process.ProcessInteractionBuilder;
-import org.arquillian.spacelift.task.os.CommandTool;
-
-@Command(name = "app-create", description = "Create OpenShift Cartridge. Requires rhc tools installed.")
```
Comment (matzew) [inline tools/test-extension/client/src/main/java/org/jboss/aerogear/test/cli/AppCreateCommand.java]: cool ! Airline!

Comment (matzew) [inline tools/test-extension/client/src/main/java/org/jboss/aerogear/test/cli/AppCreateCommand.java]: 😆

**Comment by matzew** [comment]
@kpiwko should we have README update ?

**Comment by kpiwko** [comment]
my $0.02 skip it for now, this is just enablement for guys to follow...to polish repository is the next step, with ups update as well for sure...we likely need to make it compatible with WF10 as well - so it can run in CI.

I would personally replace airline with node.js based cli and reduce few levels of abstraction Tadeas included in this repo ;-)

**Comment by kpiwko** [comment]
Rebase to be mergable to master.

**Pull request merged**